evm/cli-utils.js

```diff
@@ -1,3 +1,48 @@
+const { Option } = require('commander');
+const { addBaseOptions, ...exportedCliUtils } = require('../common/cli-utils');
+
+const addExtendedOptions = (program, options = {}) => {
```

Comment:
```suggestion
const addDeployOptions = (program, options = {}) => {
```
Maybe or `addEvmOptions`?